packs/salt/actions/lib/utils.py

```diff
@@ -28,3 +28,9 @@ def generate_action(module_type, action):
     fh.write('---\n')
     fh.write(yaml.dump(manifest, default_flow_style=False))
     fh.close()
+
+
+def sanitize_payload(keys_to_sanitize, payload):
```

Comment:
Can you throw some notes in here? I see what's happening, but future me will be very appreciative of some footprints.